tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -375,65 +375,17 @@ def run(self):
             {"message": "%s complete" % self.__class__.__name__}
         )
 
-    def _enable_disable_volume_profiling(self):
+    def _update_cluster_volume_profiling_state(self):
         cluster = NS.tendrl.objects.Cluster(
             integration_id=NS.tendrl_context.integration_id
         ).load()
```

Comment:
replace this entire cluster object load by a variable volume_profiling_state = None